codespell_lib/data/dictionary.txt

```diff
@@ -12407,6 +12415,9 @@ frankin->franklin
 Fransiscan->Franciscan
 Fransiscans->Franciscans
 franzise->franchise
+frecuent->frequent
+frecuently->frequently
```

Comment:
Can you also add a separate suggestion for the -ed version of this typo please.

Reply:
You could also add frequency (and -ies) versions of these too.

Reply:
like 5d9c9c4 ?